src/components/user/Dashboard.jsx

```diff
@@ -1,130 +1,106 @@
   const [todos, setTodos] = useState([]);
   const [todoName, setTodoName] = useState("");
 
   const getUserTodos = () => {
-    const user_id = props.user.id;
+    const userId = props.user.id;
     axios
-      .get(process.env.REACT_APP_HOST+ "/todos/show/"+user_id)
+      .get(process.env.REACT_APP_HOST + "/todos/show/" + userId)
       .then((res) => {
         if (res !== "") {
           setTodos(res.data);
         }
       })
-      .catch(() => console.error);
+      .catch();
   };
 
-
   const toggleComplete = async (id, index) => {
     const complete = todos[index].complete;
-    await axios.put(process.env.REACT_APP_HOST+"/todos/"+id, {
+    await axios.put(process.env.REACT_APP_HOST + "/todos/" + id, {
       complete: !complete,
     });
     getUserTodos();
   };
 
-  const editTaskName = async (e,id) => {
-    await axios.put(process.env.REACT_APP_HOST+"/todos/"+id, {
-      name: todoName
+  const editTaskName = async (e, id) => {
+    await axios.put(process.env.REACT_APP_HOST + "/todos/" + id, {
+      name: todoName,
     });
     getUserTodos();
   };
 
-
-  const deleteTodo = async(todoId,index) => {
+  const deleteTodo = async (todoId, index) => {
     const complete = todos[index].complete;
-    await axios.put(process.env.REACT_APP_HOST+"/todos/"+todoId, {
-      complete: !complete
+    await axios.put(process.env.REACT_APP_HOST + "/todos/" + todoId, {
+      complete: !complete,
     });
 
-    axios.delete(process.env.REACT_APP_HOST+"/todos/"+todoId)
-      .then(() => 
-      getUserTodos()
-      )
-      .catch(console.error());
+    axios
+      .delete(process.env.REACT_APP_HOST + "/todos/" + todoId)
+      .then(() => getUserTodos())
+      .catch();
   };
 
-  useEffect(() => 
-  {
+  useEffect(() => {
     props.checkLoginStatus();
     getUserTodos();
   }, [todos]);
 
   return (
     <div>
       <h1>Dashboard</h1>
-      <h2>
-        こんにちは {props.loggedInStatus ? props.user.email : "ゲスト"}　さん！
-      </h2>
-      <p>ユーザーID: {props.loggedInStatus ? props.user.id : ""}</p>
+      <HelloUser loggedInStatus={props.loggedInStatus} user={props.user} />
 
-      <h1>Todo List</h1>
-      <Task_ul>
+      <h1>Your Todo List</h1>
+      <ul>
         {todos.map((todo, index) => (
           <List size="xl">
             <List.Item>
-              <Task_li index={index}>
-                <TaskName_div>
+              <TaskList index={index}>
+                <TaskNameDiv>
                   <TaskName>
                     {todo.complete ? <s>{todo.name}</s> : todo.name}
                   </TaskName>
-                  {/* <button  onClick={() => getTodosUser(todo,index)}>who</button> */}
-                  {/* <p>user_id: {todo.user_id}</p> */}
-                </TaskName_div>
+                </TaskNameDiv>
 
-                <Button_li>
-                  <Edit_div
+                <ButtonList>
+                  <div
                     style={{ display: todo.complete ? "none" : "block" }}
```

Comment:
todo.completeはbooleanだと思うのでこれでも良さそう？
```suggestion
                    style={{ display: todo.complete }}
```